proto/stat.proto

```diff
@@ -46,16 +46,20 @@ message PointStat {
   // date in IOS-8601 format
   string date = 1;
   double value = 2;
-  // Full metadata of the stat
+  // Full metadata of the stat. This is set when the proto is used statalone,
+  // ex, in place page cache.
   StatMetadata metadata = 3;
   // Hash of the metadata, this is to be used together with a map from
-  // the hash to the full metatdata.
+  // the hash to the full metatdata. This is set in /stat/set/within-place/*
```

Comment:
nit: It doesn't apply to all right? maybe worth being precise on what APIs set this.